quadriga/src/main/java/edu/asu/diging/quadriga/core/model/EventGraph.java

```diff
@@ -1,19 +1,29 @@
 package edu.asu.diging.quadriga.core.model;
 
+import java.time.LocalDateTime;
+
 import org.bson.types.ObjectId;
 
+import edu.asu.diging.quadriga.api.v1.model.Context;
 import edu.asu.diging.quadriga.core.model.events.CreationEvent;
 
 public class EventGraph {
 
     private ObjectId _id;
     private CreationEvent  rootEvent;
     private DefaultMapping defaultMapping;
+    private LocalDateTime creationTime;
+    private ObjectId collectionId;
+    private String appName;
+    private Context context;
     
-    public EventGraph() {}
+    public EventGraph() {
+        this.creationTime = LocalDateTime.now();
```

Comment:
the creation time would come from the database, so it shouldn't be set here.

Reply:
Umm I am not sure I understand
So when a network is submitted, we add a new EventGraph entry in the database
The database doesn't have the EventGraph, we're adding a new one, not finding it from the DB

Did you mean that the creation time would come from the JSON request for the network submission?